lars-reimann/api-editor
# feat: Creation of test files for migration of annotations

Closes #1181.

### Summary of Changes

- Add two versions of an api, its api data and (migrated) annotation data
- Improve detection of instance attributes


### Testing Instructions

run the parse-package commands for the apis and the annotations and verify that the json files are correct generated or use the script in `migration_test_data`
